packages/template-retail-react-app/app/components/passwordless-login/index.jsx

```diff
@@ -32,6 +34,11 @@ const PasswordlessLogin = ({
         }
     }
 
+    const updateSubmittedEmailRef = () => {
+        console.log('hellooooooooo')
+        submittedEmail.current = document.getElementById('email').value
```

Comment:
🤔 seeing "document" access here gives me some code smell vibes. Maybe lets talk to discuss what it is you are trying to achieve and whether or not this is the correct react-y way of doing it.

Reply:
Removed!